src/components/chart/_chart.scss

```diff
@@ -0,0 +1,22 @@
+.ons-chart {
+    margin: 0;
+    &__download-title {
+        @extend .ons-u-pt-l;
+        @extend .ons-u-fs-r--b;
+    }
+    &__title {
+        font-size: 1.25rem;
```

Comment:
@precious-onyenaucheya-ons are these different sizes for the usual heading 2, heading 3 etc in the design system? While it makes sense to use the data vis guidelines for the axis labels and titles, I wonder if we ought to match the standard sizes for headings, otherwise this will appear different to other headings on the page.

Reply:
@helenb Yeah, I agree. However, these sizes are based on the data visualization guidelines, which we’ve been advised to follow.

Reply:
Discussed this with @precious-onyenaucheya-ons and agreed to leave it at the data vis sizes for now as it is easily updated later.